Stellar-transaction.x

```diff
@@ -578,6 +579,22 @@ struct InvokeHostFunctionOp
     SorobanAuthorizationEntry auth<>;
 };
 
+enum BumpExpirationType
+{
+    BUMP_EXPIRATION_UNIFORM = 0
+};
+
+/* Bump the expiration ledger of the entries specified in the readOnly footprint
+   so they'll expire at least ledgersToExpire ledgers from lcl.
+    Threshold: low
```

Comment:
Doesn't this op allow spending potentially high amounts of account funds? I'd say it should probably be medium - it's not that different from a payment...

Reply:
I made this low because InvokeHostFunctionOp is also low. Should they both be medium instead?

Reply:
Hmm, that's a bug then. Invoke host fn should definitely be medium.

Reply:
The [CAP](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0046-04.md) specifically mentions that InvokeHostFunctionOp should be low, but I don't remember Jon's reason for it (I do remember discussing this with him though).

Reply:
That doesn't really make any sense now (maybe it did in pre-SAC world). Payment is medium threshold and we don't want to relax that with SAC.

Reply:
Hmm what you're saying makes sense. I'm think this made sense because account auth in the contract used to default to medium threshold back then. I'll make the change here and in core.